### PR TITLE
Adding Korean Translation - 1st pull request

### PR DESCRIPTION
**Pull request by jousinKaka**
This is a Korean translation of Clean Code, which I first started as an assignment.

If there is anything that needs to be changed, please let me know.

I sincerely hope that this will add '한국어' to the cleanabap.md file.

### CODE REVIEW

File: clean-abap/CleanABAP.md [hunk 1]
```diff
@@ -297,29 +300,29 @@ where mixing styles could cause confusion, for example:
 - Mixing `REF TO` and `FIELD-SYMBOL` when looping.
 - Mixing `NEW` and `CREATE OBJECT` when calling a `CONSTRUCTOR`.
 - Mixing `RETURNING` and `EXPORTING` in the method signatures of
-methods only returning / exporting one parameter.
+  methods only returning / exporting one parameter.
```
Comment (xtough) [inline clean-abap/CleanABAP.md:303]: Unnecessary change of indentation.

File: clean-abap/CleanABAP.md [hunk 2]
```diff
@@ -297,29 +300,29 @@ where mixing styles could cause confusion, for example:
...
 
 We observed good results with a four-step plan for refactoring:
 
 1. Get the team aboard. Communicate and explain the new style,
-and get everybody on the project team to agree to it.
-You don't need to commit all guidelines at once, just start
-with an undisputed small subset and evolve from there.
+   and get everybody on the project team to agree to it.
```
Comment (xtough) [inline clean-abap/CleanABAP.md:308]: Unnecessary change of indentation.

File: clean-abap/CleanABAP_es.md [hunk 3]
```diff
@@ -2,21 +2,26 @@
...
+> &nbsp;·&nbsp;
+> [한국어](../../styleguides4/clean-abap/CleanABAP_kr.md)
+
 
 
 Esta guía fue adaptada en base a [_Clean Code_ por Robert C. Martin] 
 para [ABAP](https://en.wikipedia.org/wiki/ABAP).
 
-La [Cheat Sheet](cheat-sheet/CheatSheet.md) es una versión optimizada para impresión.
+
```
Comment (xtough) [inline clean-abap/CleanABAP_es.md:22]: Unnecessary whitespace

File: clean-abap/CleanABAP_fr.md [hunk 4]
```diff
@@ -24,221 +27,221 @@ L'[antisèche](cheat-sheet/CheatSheet.md) est une version optimisée pour l'impr
 ## Table des matières
 
 - [Comment faire pour](#comment-faire-pour)
-   - [Démarrer avec la méthodologie Clean Code](#démarrer-avec-la-méthodologie-clean-code)
-   - [Refactoriser du code existant](#refactoriser-du-code-existant)
-   - [Effectuer une vérification automatique](#effectuer-une-vérification-automatique)
-   - [Faire le lien avec d'autres guides](#faire-le-lien-avec-dautres-guides)
-   - [Exprimer un désaccord](#exprimer-un-désaccord)
+    - [Démarrer avec la méthodologie Clean Code](#démarrer-avec-la-méthodologie-clean-code)
```
Comment (xtough) [inline clean-abap/CleanABAP_fr.md:30]: same here

**Review by xtough — CHANGES_REQUESTED** [review]
Apart from your main contribution, clean-abap/CleanABAP_kr.md, could you please concentrate on the absolutely necessary changes in the other files? 
I have noticed that you have unnecessarily increased indentation or inserted many whitespace characters in almost all files. This makes it hard to review and rebase your change.

**Review by jousinKaka — COMMENTED** [review]
I just want to add file(cleanabap_kr.md), and  just add these sentenses

'
> &nbsp;·&nbsp;
> [한국어](CleanABAP_kr.md)
'

I didn't want to touch other things, so I deleted only the conflict one. Is there any way to get it back?

**Review by jousinKaka — COMMENTED** [review]
Please Check it. I've modified them.

**Review by xtough — APPROVED** [review]
looks good to me :-)
